src/hooks/useSparkleFirestoreConnect.ts

```diff
@@ -41,7 +41,7 @@ export interface SparkleRFQConfig extends ReduxFirestoreQuerySetting {
  * @see ReduxFirestoreQuerySetting
  */
 export const useSparkleFirestoreConnect = (
-  config?: SparkleRFQConfig[] | SparkleRFQConfig
+  config?: SparkleRFQConfig[] | SparkleRFQConfig | (() => SparkleRFQConfig[])
```

Comment:
I was thinking about adding a ticket to the backlog to have it like this across the whole codebase

Reply:
Definitely worth adding a backlog ticket for converting our 'ternary' patterns to this. If we still have 'raw' queries, it wouldn't make sense to wrap them in a function just for the sake of it.

I'm sure it will be fine, as in other places I checked `useFirestoreConnect` does a deep comparison for it's memoization.. but I would want to confirm that we wouldn't break any memo things by using non-stable function signatures before we change things on a large scale.

Worth adding these comments to the backlog issue when you create it.

Reply:
https://github.com/sparkletown/sparkle/issues/1069

Reply:
Transferred that issue to https://github.com/sparkletown/internal-sparkle-issues/issues/136 as you made it on the public repo.